src/js/tabs/setup.js

```diff
@@ -416,35 +415,58 @@ setup.initialize = function (callback) {
             // Firmware info
             msp_api_e.text([FC.CONFIG.apiVersion]);
             build_date_e.text([FC.CONFIG.buildInfo]);
 
-            if (FC.CONFIG.buildKey.length === 32) {
-                const buildRoot   = `https://build.betaflight.com/api/builds/${FC.CONFIG.buildKey}`;
-                const buildConfig = `<span class="buildInfoBtn" title="${i18n.getMessage('initialSetupInfoBuildInfoConfig')}: ${buildRoot}/json">
-                                     <a href="${buildRoot}/json" target="_blank"><strong>${i18n.getMessage('initialSetupInfoBuildInfoConfig')}</a></strong></span>`;
-                const buildLog =    `<span class="buildInfoBtn" title="${i18n.getMessage('initialSetupInfoBuildInfoLog')}: ${buildRoot}/log">
-                                     <a href="${buildRoot}/log" target="_blank"><strong>${i18n.getMessage('initialSetupInfoBuildInfoLog')}</a></strong></span>`;
-                build_info_e.html(`${buildConfig} &nbsp &nbsp ${buildLog}`);
-                $('.build-info a').removeClass('disabled');
-            } else {
-                $('.build-info a').addClass('disabled');
-            }
+            if (navigator.onLine) {
+                let buildOptionList = "";
 
-            if (FC.CONFIG.buildOptions.length) {
-                let buildOptions = "";
-                build_opt_e.text = "";
+                if (FC.CONFIG.buildOptions.length) {
 
-                for (const buildOption of FC.CONFIG.buildOptions) {
-                    buildOptions = `${buildOptions} &nbsp ${buildOption}`;
+                    for (const buildOptionElement of FC.CONFIG.buildOptions) {
+                        buildOptionList = `${buildOptionList} &nbsp ${buildOptionElement}`;
+                    }
+                }
+
+                if (FC.CONFIG.buildKey.length === 32) {
+                    const buildRoot   = `https://build.betaflight.com/api/builds/${FC.CONFIG.buildKey}`;
+                    const buildConfig = `<span class="buildInfoBtn" title="${i18n.getMessage('initialSetupInfoBuildConfig')}: ${buildRoot}/json">
+                                         <a href="${buildRoot}/json" target="_blank"><strong>${i18n.getMessage('initialSetupInfoBuildConfig')}</strong></a></span>`;
+
+                    const buildLog =    `<span class="buildInfoBtn" title="${i18n.getMessage('initialSetupInfoBuildLog')}: ${buildRoot}/log">
+                                         <a href="${buildRoot}/log" target="_blank"><strong>${i18n.getMessage('initialSetupInfoBuildLog')}</strong></a></span>`;
+
+                    const buildOptions = `<span class="buildInfoBtn" title="${i18n.getMessage('initialSetupInfoBuildOptionList')}">
+                                         <a class="buildOptions disabled" href=#"><strong>${i18n.getMessage('initialSetupInfoBuildOptions')}</strong></a></span>`;
+
+                    build_info_e.html(`${buildConfig} ${buildLog} &nbsp &nbsp ${buildOptions}`);
```

Comment:
I believe &nbsp; should end with semicolon:
```suggestion
                    build_info_e.html(`${buildConfig} ${buildLog} &nbsp; &nbsp; ${buildOptions}`);
```